docs/clearml_agent.md

```diff
@@ -184,6 +184,18 @@ You can spin up an agent on any machine: on-prem and/or cloud instance. When spi
 service a queue(s). Utilize the machine by enqueuing tasks to the queue that the agent is servicing, and the agent will 
 pull and execute the tasks. 
 
+:::tip cross-platform execution
+ClearML Agent can be executed cross-platform. To do so, make sure to define platform-specific flags in the environment
```

Comment:
@pollfly "ClearML agent is platform agnostic. When using the ClearML Agent to execute experiments cross-platform, set platform specific environment variables before your launch the agent"